### DISCUSSION TITLE
Help me find a bypass for ContentKeeper

### DISCUSSION THREAD

**Discussion by Gumkid3** [3kh0/ext-remover#314] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need help finding a bypass for this blocker called ContentKeeper. so far, I have not found much yet. if you find any bypasses/websites, plz comment them.

**Comment by Aka-but-nice** [comment, marked as answer]
Yo, so make a bookmark with chrome://settings/signOut make sure, the o in signOut is capital, then make another bookmark with chrome://restart and click the chrome://settings/signOut first  and press then press the restart one.

**Reply by Aka-but-nice** [reply]
BTW this is not mine 
who ever has the nuke don't put it at my house!
